listenbrainz/webserver/static/js/src/utils/utils.tsx

```diff
@@ -472,10 +472,13 @@ const getAlbumArtFromListenMetadata = async (
       );
       if (CAAResponse.ok) {
         const body: CoverArtArchiveResponse = await CAAResponse.json();
-        if (!body.images?.[0]?.thumbnails) {
```

Comment:
CAA has a `/front` API endpoint which would do this for us, does it make sense to just use this endpoint instead of searching for a front from all of the returned images? https://musicbrainz.org/doc/Cover_Art_Archive/API#.2Frelease.2F.7Bmbid.7D.2Ffront